docs/UserGuide.md

```diff
@@ -1,29 +1,9 @@
----
-layout: page
-title: User Guide
----
```

Comment:
We need these few lines to indicate page's meta. It would be good if we can add
```
---
layout: page
title: User Guide
---
```
back to user guide

Reply:
Noted and updated accordingly.